charts/seed-bootstrap/templates/grafana/grafana-deployment.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: grafana
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: monitoring
+    component: grafana
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      component: grafana
+  template:
+    metadata:
+      labels:
```

Comment:
You need checksum annotations for `grafana-dashboards`, `grafana-datasources`, `grafana-dashboard-providers` config maps